src/utils/details.h

```diff
@@ -23,7 +23,18 @@
  * <https://mrcpp.readthedocs.io/>
  */
 
+#include <array>
+#include <iostream>
+
 namespace mrcpp {
+ 
+template <typename T, int D> inline auto operator<<(std::ostream &os, const std::array<T, D> &coord) -> std::ostream & {
+    os << "[";
+    for (auto elem : coord) { os << elem << ", "; }
+    os << "]";
```

Comment:
Won't this get the annoying trailing comma?
```
[a, b, c,]
```

Reply:
The obnoxious comma is gone. :boom: